src/stage0.txt

```diff
@@ -15,7 +15,7 @@
 date: 2019-09-25
 rustc: beta
 cargo: beta
-rustfmt: beta
+rustfmt: nightly
```

Comment:
We'll want to instead update rustfmt (and perhaps backport a patch to beta or something, to avoid waiting) to have something like `RUSTFMT_BOOTSTRAP` which allows us to bypass stability checking in-tree.

Reply:
OK, I think I follow. Will check in on it if I have issues.

Reply:
Opened https://github.com/rust-lang/rustfmt/pull/3900 for discussion & review.